samples/analyze.v1p3beta1.js

```diff
@@ -0,0 +1,362 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function detectPerson(path) {
+  //[START video_detect_person_beta]
+  // Imports the Google Cloud Video Intelligence library + Node's fs library
+  const Video = require('@google-cloud/video-intelligence').v1p3beta1;
+  const fs = require('fs');
+  // Creates a client
+  const video = new Video.VideoIntelligenceServiceClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const path = 'Local file to analyze, e.g. ./my-file.mp4';
+
+  // Reads a local video file and converts it to base64
+  const file = fs.readFileSync(path);
+  const inputContent = file.toString('base64');
+
+  const request = {
+    inputContent: inputContent,
+    features: ['PERSON_DETECTION'],
+    videoContext: {
+      personDetectionConfig: {
+        // Must set includeBoundingBoxes to true to get poses and attributes.
+        includeBoundingBoxes: true,
+        includePoseLandmarks: true,
+        includeAttributes: true,
+      },
+    },
+  };
+  // Detects people in a video
+  const [operation] = await video.annotateVideo(request);
+  const results = await operation.promise();
+  console.log('Waiting for operation to complete...');
+
+  // Gets annotations for video
+  const personAnnotations =
+    results[0].annotationResults[0].personDetectionAnnotations;
+
+  for (const {tracks} of personAnnotations) {
+    console.log('Person detected:');
+    for (const {segment, timestampedObjects} of tracks) {
+      if (segment.startTimeOffset.seconds === undefined) {
+        segment.startTimeOffset.seconds = 0;
+      }
+      if (segment.startTimeOffset.nanos === undefined) {
+        segment.startTimeOffset.nanos = 0;
+      }
+      if (segment.endTimeOffset.seconds === undefined) {
+        segment.endTimeOffset.seconds = 0;
+      }
+      if (segment.endTimeOffset.nanos === undefined) {
+        segment.endTimeOffset.nanos = 0;
+      }
```

Comment:
@telpirion, does this ever happen? 
If so, that seems like an issue we need to raise.

Reply:
It can happen, yes. Sometimes the result from the API simply doesn't return a value and thus the property is undefined.

TBH, I'm following the pattern demonstrated here (among other places):
https://github.com/googleapis/nodejs-video-intelligence/blob/master/samples/analyze.v1p2beta1.js#L42-L61